eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1545,9 +1545,21 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
                 ()
             }
           }
+
           // we will re-enable the channel after some delay to prevent flappy updates in case the connection is unstable
           setTimer(Reconnected.toString, BroadcastChannelUpdate(Reconnected), 10 seconds, repeat = false)
 
+          // We usually handle feerate updates once per block (~10 minutes), but when our remote is a mobile wallet that
```

Comment:
What do you think? A bit simpler to reason about no?

Reply:
Yep looks good and it's not much code.